public_html/layouts/basic/modules/Calendar/resources/Extended/CalendarView.js

```diff
@@ -758,7 +758,7 @@ window.Calendar_CalendarExtended_Js = class Calendar_CalendarExtended_Js extends
 				startDate = startDate + 'T' + start_hour + ':00';
 				endDate = endDate + 'T' + end_hour + ':00';
 				if (startDate == endDate) {
-					let activityType = data.find('[name="activitytype"]').val();
+					let activityType = data.find('[name="activitytype"]:checked').val();
```

Comment:
this will not work for:
https://github.com/YetiForceCompany/YetiForceCRM/blob/4b388f4747941c7d6cfabd1569ed70e49bd49a3c/modules/Calendar/ConfigTemplate.php#L148-L150